### ISSUE TITLE
How do i add encrypted files or env variables to the build process?

### ISSUE THREAD

**Issue by frozeman** [EducationLink/meteor-google-cloud#16]
I followed the [tutorial here](https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-secrets-credentials), but using this package I couldn't add the `cloudbuild.yml` anywhere, as it doesn't use the root folder to deploy, but the `bundle` folder. How could it add the decrypt steps without forking your package?

**Comment by iamstevendao** [comment]
@frozeman You can use option `--build-only`, which will build the project into `output/bundle` folder, then you can deploy from there.

**Comment by frozeman** [comment]
i see thanks.

So simply 

```bash
$ ... --build-only
$ cd output/bundle
$ gcloud submit ....
```


But it would be nice to be able to simply add google cloud build steps to decode ENV or files while submitting.

**Comment by iamstevendao** [comment]
@frozeman Yep, it would be nice. For now, if you used cloud build steps to decrypt the files, `meteor-google-cloud` would be used for the build step only.

**Comment by raphaelarias** [comment]
I tend to believe the package should take care of the build and deploy only. If we add decryption of variables it may start to increase the scope too much. I'm not sure.
